pennylane/workflow/qnode.py

```diff
@@ -592,8 +592,17 @@ def _update_gradient_fn(self, shots=None, tape=None):
             return
         if tape is None and shots:
             tape = qml.tape.QuantumScript([], [], shots=shots)
+
+        diff_method = self.diff_method
+        if (
+            self.device.name == "lightning.qubit"

```

Comment:
quick question but do we need to validate against `lightning.kokkos` and `lightning.gpu` as well? Or do they not yet default to adjoint?

Reply:
They do not yet default to adjoint.